tasks/task07/unittests.py

```diff
@@ -0,0 +1,42 @@
+#tests for Task 07: OrderRepository
```

Comment:
it is common to name test files like `test_{subject_file}`, i.e. `test_order_repository.py`